templates/hooks/db_instance/sdk_file_end.go.tpl

```diff
@@ -1,7 +1,9 @@
 {{ $CRD := .CRD }}
 {{ $SDKAPI := .SDKAPI }}
 
-{{- $operation := (index $SDKAPI.API.Operations "RestoreDBInstanceFromDBSnapshot")}}
+{{ range $operationName := Each "RestoreDBInstanceFromDBSnapshot" "CreateDBInstanceReadReplica" }}
```

Comment:
++ clean

Reply:
can you elaborate a little bit?

Reply:
Just saying this code is clean. No action here